backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.adoc

```diff
@@ -32,6 +32,11 @@ To back up PVs with snapshots, you must have a cloud provider that supports eith
 * xref:../../../backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp.adoc#installing-oadp-gcp[Google Cloud Platform]
 * CSI snapshot-enabled cloud provider, such as xref:../../../backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc#installing-oadp-ocs[OpenShift Data Foundation]
 
+[NOTE]
+====
+You can configure multiple backup storage locations within the same namespace for each individual OADP deployment.
```

Comment:
Is this note specific to the CSI snapshot? If not, then it might be better in a different location. 

The snippet below looks to be specific to CSI snapshots, so probably should be kept right beneath it.

Also, it's usually best ([per SSG here](https://redhat-documentation.github.io/supplementary-style-guide/#admonitions)) to avoid putting notes back-to-back. 

I'd see if there is a better spot in the document for this note. Or it really it seems like it might not need to be a note at all, and could just be a sentence added to the regular text somewhere.